src/MockClassifier.Api/Program.cs

```diff
@@ -21,7 +22,9 @@ public static void Main(string[] args)
 
             // Add services to the container.
             var dmrSettings = configuration.GetSection("DmrServiceSettings").Get<DmrServiceSettings>();
-            services.AddDmrService(dmrSettings);
+            var centOpsSettings = configuration.GetSection("DmrServiceSettings").Get<CentOpsServiceSettings>();
```

Comment:
See comment in this PR: https://github.com/buerokratt/Mock-Bot/pull/40/files#r925385206

I think the same thing applies here too.